bin/cron/create_rollup_tables

```diff
@@ -144,8 +144,12 @@ def main
   end
 
   # Verify that the data size is sensible.
-  raise "contained_levels too big: #{contained_levels.size} rows exceeds limit of 10000" if contained_levels.size > 10_000
-  raise "contained_level_answers too big: #{contained_levels_answers.size} rows exceeds limit of 100000" if contained_level_answers.size > 100_000
+  CDO.log.info "contained_levels: #{contained_levels.size}"
+  CDO.log.info "contained_level_answers: #{contained_level_answers.size}"
+  CDO.log.info "level_sources: #{level_sources_multi_types.size}"
```

Comment:
Just for my knowledge -- and I should know this -- where can we see these logs?

Reply:
I will also need to figure this out 😅

Reply:
they're in cloudwatch log groups: https://codedotorg.slack.com/archives/C0T0PNTM3/p1684520216970339